docs/glossary.md

```diff
@@ -189,6 +189,14 @@ Tickets that reached the end of their window without being called to vote - thes
 
 A mechanism for obtaining free (testnet) coins.
 
+#### Fork
+
+A programming term for an open-source code modification. The modified code is considered a “fork” of the original code. Typically, forks contain modifications that significantly alter the original code. For example, the Bitcoin Cash fork of the Bitcoin blockchain raised the block size from 1Mb to 8Mb. If the state of the blockchain is also forked, as was the case with Bitcoin Cash, it is also considered a “chain fork”. In this case, owners of coins on the original chain will have equivalent coins on the new chain when the chains diverge. Decred began its life as a source code fork of [btcd](https://github.com/btcsuite/btcd), an alternate full node implementation of Bitcoin. Significant changes were then made before launch, including modifications to the consensus rules, a Proof-of-Stake (PoS) layer to enable on-chain governance, a new hashing algorithm and other innovations. It is arguable whether Decred is a source code fork, considering the scope of modifications made. Decred is not a chain fork, as it did not copy the state of the Bitcoin blockchain, and instead started from its own genesis block. 
```

Comment:
Decred code is a fork of btcsuite. I do not think this is arguable, I think this is a fact. Fork is a well defined term - using a copy of the source code from one project as the basis of another project - and that is exactly what happened with btcsuide and Decred.

Reply:
Curious for your opinion on this @davecgh?

Reply:
It would probably be most accurate to say that Decred began its life as a source code fork of [btcsuite](https://github.com/btcsuite), because more than just dcrd/btcd required modifications, but, the core network is indeed dcrd, so I didn't bother nitpicking that on my review.

In terms of the bit in question here about it being arguable, I agree from a purely technical standpoint, but I also understand what the text is conveying, so it didn't really stand out as something major to me.   The issue is that typically when you talk about something being a fork of something else, it gives the impression that the are very few differences between the two, which is really not the case here at all.  The two codebases have _significantly_ diverged at this point that still referring to it as such doesn't really properly describe the situation.

Perhaps something like "...were then made before launch, including ... a new hashing algorithm and other innovations.  Additionally, since launch, the code base has continued to diverge from btcsuite so significantly that it is a standalone code base."

Reply:
Upon rereading, I see @jholdstock's point. I think the fact that we say "Decred **began its life** as a source code fork of [btcd](https://github.com/btcsuite/btcd)," and then describe the significant changes, is enough to make the argument that it is its own standalone code base now. We could keep wordsmithing this, but this entry is getting long already, and I think it's better to be concise. Here's what I have now:

>Decred began its life as a source code fork of [btcd](https://github.com/btcsuite/btcd), an alternate full node implementation of Bitcoin. Significant changes were then made before launch, including modifications to the consensus rules, a Proof-of-Stake (PoS) layer to enable on-chain governance, a new hashing algorithm and other innovations. Decred is not a chain fork, as it did not copy the state of the Bitcoin blockchain, and instead started from its own genesis block.